pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -15,15 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	DNSName string `json:"name,omitempty"`
-	PktType string `json:"pktType,omitempty"`
-	QType   string `json:"qtype,omitempty"`
+	PktType string `json:"pktType,omitempty" column:"type,width:9,fixed"`
+	QType   string `json:"qtype,omitempty" column:"qtype,width:10,fixed"`
+	DNSName string `json:"name,omitempty" column:"name,width:24"`
```

Comment:
What do you think about using 30 (Same as k8s columns)? In this way, they will be autoscaled identically. Current situation:

```bash
$ ./kubectl-gadget trace dns -A
NODE                      NAMESPACE                 POD                       TYPE      QTYPE      NAME
worker                                                                        OUTGOING  A          registry-1.docker.i…
worker                                                                        HOST      A          registry-1.docker.i…
master                    kube-system               coredns-78fcd69978-rj79d  HOST      AAAA       www.inspektor-gadge…
```

Reply:
I agree. Let's give it more "weight".